index.js

```diff
@@ -156,8 +159,14 @@ function processContains(/* CODE HERE */) {
  * [2] Invoking `processDuplicateFree` passing `[1,1,2,2,3]` and `(arr) => arr.length`,
  * should return 3.
 */
-function processDuplicateFree(/* CODE HERE ONLY AFTER COMPLETING ALL OTHER TASKS */) {
-  /* CODE HERE ONLY AFTER COMPLETING ALL OTHER TASKS */
+function processDuplicateFree(list, callback) {
+  let deDuped = [];
+  for(let i=0; i< list.length; i++){
+    if(deDuped.includes(list[i])===false){
```

Comment:
This one way to do it . Another is what we discussed 
`if(!deDuped.includes(list[i]))`